src/cmap/stream_description.ts

```diff
@@ -34,20 +42,22 @@ class StreamDescription {
         : [];
   }
 
-  receiveResponse(response: any) {
+  receiveResponse(response: Document): void {
     this.type = parseServerType(response);
-    RESPONSE_FIELDS.forEach((field: any) => {
+    RESPONSE_FIELDS.forEach(field => {
       if (typeof response[field] !== 'undefined') {
-        (this as any)[field] = response[field];
+        this[field] = response[field];
+      }
+
+      // testing case
+      if ('__nodejs_mock_server__' in response) {
```

Comment:
I removed the casting but it is still in a separate statement because all the other keys in that list are of type number, but this is a boolean so you get a union type for `response[field]` that doesn't assign to `this[field]`

Reply:
cool, hopefully we can get rid of that field completely in the future